doc/classes/Performance.xml

```diff
@@ -302,5 +309,17 @@
 		<constant name="MONITOR_MAX" value="59" enum="Monitor">
 			Represents the size of the [enum Monitor] enum.
 		</constant>
+		<constant name="MONITOR_TYPE_QUANTITY" value="0" enum="MonitorType">
+			Monitor output is formatted as an integer value.
+		</constant>
+		<constant name="MONITOR_TYPE_MEMORY" value="1" enum="MonitorType">
+			Monitor output is formatted as computer memory, submitted values should represent number of bytes.
+		</constant>
+		<constant name="MONITOR_TYPE_TIME" value="2" enum="MonitorType">
+			Monitor output is formatted as time in milliseconds, submitted values should represent time in seconds.
```

Comment:
It's probably worth clarifying just in case:

```suggestion
			Monitor output is formatted as time in milliseconds. Submitted values should represent a time in seconds (not milliseconds).
```